mygithub/src/components/github/Profile.js

```diff
@@ -1,58 +1,59 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import RepoList from './RepoList.js';
-class Profile extends Component{
-	
-	render(){
-// Nesting profile component in App component
-		return(
+class Profile extends Component {
+
+	render() {
+		// Nesting profile component in App component
+		return (
 			<div>
 				<div className="panel panel-default">
-					  <div className="panel-heading"></div>
-					  <div className="panel-body">
-					    <h1> Name :{this.props.userData.name} </h1>
-					  </div>
+					<div className="panel-heading"></div>
+					<div className="panel-body">
+						<h1> Name :{this.props.userData.name} </h1>
+						<h1> Name :{this.props.userData.address} </h1>
+					</div>
 				</div>
 
 				<div className="panel panel-default">
-					  <div className="panel-heading">
-					  <h3 className="panel-title">Github Profile</h3>
-					  </div>
-					  <div className="panel-body">
-					    <div className ="row">
-						    <div className="col-md-4">
-						    <img src ={this.props.userData.avatar_url} className ="thumbnail" style ={{width:"50%"}}/>
+					<div className="panel-heading">
+						<h3 className="panel-title">Github Profile</h3>
+					</div>
+					<div className="panel-body">
```

Comment:
Excellent use of inline styles to control the width of the image! This enhances the component's visual appeal. However, for more extensive styling, consider using a separate CSS file for better organization and to avoid mixing presentation with logic.